src/doc/rustdoc/src/the-doc-attribute.md

```diff
@@ -0,0 +1,108 @@
+# The `#[doc]` attribute
+
+The `#[doc]` attribute lets you control various aspects of how `rustdoc` does
+its job. 
+
+The most basic job of `#[doc]` is to be the way that the text of the documentation
+is handled. That is, `///` is syntax sugar for `#[doc]`. This means that these two
+are the same:
+
+```rust,ignore
+/// This is a doc comment.
+#[doc = "This is a doc comment."]
```

Comment:
Note that these are not actually the same! The former has a leading space, but the latter does not. This is why the `unindent-comments` pass exists, and also is the reason #42760 exists.